data/migrations/env.py

```diff
@@ -128,48 +97,20 @@ def run_migrations_online():
 
     In this scenario we need to create an Engine and associate a connection with the context.
     """
-
-    if isinstance(db.obj, SqliteDatabase) and not "DB_URI" in os.environ:
-        print("Skipping Sqlite migration!")
+    if isinstance(db.obj, SqliteDatabase) and "DB_URI" not in os.environ:
+        logger.info("Skipping Sqlite migration!")
         return
 
-    progress_reporter = get_progress_reporter()
-    context.config.attributes["progress_reporter"] = progress_reporter
-    op = ProgressWrapper(alembic_op, progress_reporter)
-
-    migration = Migration()
-    version_apply_callback = report_success
-    if truthy_bool(
-        context.get_x_argument(as_dictionary=True).get("generatedbaopmigrations", False)
-    ):
-        op = OpLogger(alembic_op, migration)
-        version_apply_callback = partial(finish_migration, migration)
-
-    engine = engine_from_config(
-        config.get_section(config.config_ini_section), prefix="sqlalchemy.", poolclass=pool.NullPool
-    )
-
-    revision_to_migration = {}
-
-    def process_revision_directives(context, revision, directives):
-        script = directives[0]
-        migration = Migration()
-        revision_to_migration[(script.rev_id, revision)] = migration
-        migration.add_hints_from_ops(script.upgrade_ops)
-
+    engine = get_engine()
```

Comment:
Will this know where to look for migrations? We were previously getting it from the `ini` file (`data/migrations`).

Reply:
Yeah. It still looks in the right place to apply migrations and also uses the correct models (metadata) when generating new migrations.